oceanprotocol/status-monitoring-service
# Split up Status Service

Fixes #2 

Changes proposed in this PR:

- The existing status service has been split into 2 parts. This part runs the monitoring service on a cron job. It then calls the API to update the DB.

## To test:

- first clone https://github.com/oceanprotocol/status-api, set up .env and then `npm i && npm run dev`
- next clone this repo (on this branch), set up `.env` and then `npm i && npm run dev`. The cron job will then start and the results will be set to the DB via the API.
- The tests should also work in both repos.
